components/pkg-export-helm/src/main.rs

```diff
@@ -84,17 +98,27 @@ fn cli<'a, 'b>() -> clap::App<'a, 'b> {
         .arg(
             Arg::with_name("VERSION")
                 .value_name("VERSION")
+                .short("v")
                 .long("version")
                 .validator(valid_version)
-                // Keep the default version here in sync with chartfile::DEFAULT_VERSION
-                .help("Version of the chart to create (default: 0.0.1)"),
+                .help(&CHART_VERSION_HELP),
         )
         .arg(
             Arg::with_name("DESCRIPTION")
                 .value_name("DESCRIPTION")
                 .long("desc")
                 .help("A single-sentence description"),
         )
+        .arg(
+            Arg::with_name("OPERATOR_VERSION")
+                .value_name("OPERATOR_VERSION")
+                .long("operator-version")
+                .validator(valid_version)
+                .help(&OPERATOR_VERSION_HELP),
+        )
+        .arg(Arg::with_name("DOWNLOAD_DEPS").long("download-deps").help(
+            "Whether to download dependencies (default: no)",
```

Comment:
> Whether to download dependencies (default: no)

Think that could use rephrasing? Also maybe explain what kind of dependencies here, especially for new Helm users.

Reply:
You can use [`default_value`](https://docs.rs/clap/2.30.0/clap/struct.Arg.html#method.default_value) to set defaults in clap.